opensrp-chw/src/ba/java/org/smartregister/chw/activity/FamilyOtherMemberProfileActivityFlv.java

```diff
@@ -28,9 +26,8 @@ public OnClickFloatingMenu getOnClickFloatingMenu(final Activity activity, final
                     FamilyCallDialogFragment.launchDialog(activity, familyBaseEntityId);
                     break;
                 case R.id.refer_to_facility_layout:
-                    if (BuildConfig.USE_UNIFIED_REFERRAL_APPROACH) {
-                        addReferralTypes(activity);
-                        org.smartregister.chw.util.Utils.launchClientReferralActivity(activity, referralTypeModels, baseEntityId);
+                    if (BuildConfig.USE_UNIFIED_REFERRAL_APPROACH) {                       ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426619)